creature.py

```diff
@@ -1,13 +1,19 @@
 import random
+from weapon import *
+
 
 class Creature:
     id = 1
 
     def __init__(self):
-        self.health = 50
+        self.health = 25
         self.name = f"creature_{Creature.id}"
+        self.ac = 11
         self.exp = 0
         self.lvl = 1
+        self.weapon = weapons[random.randint(0, (len(weapons) - 1))]
+        self.dmg = self.weapon.damage()
+        self.what_weapon = self.weapon.name
```

Comment:
Bad name for a field. Field should have name with some statement without question.

Reply:
Either `weapon_name` should work or you just re-use `self.weapon.name` field where it's needed. Why duplicate information? What if you change the `weapon` but forget to change `self.weapon_name`? You can also define property `weapon_name` if it's necessary.